frontend/lib/ui/KeyboardKeys/KeyboardKeys.tsx

```diff
@@ -26,6 +26,18 @@ function renderKey(keyboardKey: KeyboardKey, index: number): React.JSX.Element {
           <span>Shift</span>
         </kbd>
       );
+    case KeyboardKey.Down:
+      return (
+        <kbd key={index}>
+          <span>↓</span>
+        </kbd>
+      );
+    case KeyboardKey.Up:
+      return (
+        <kbd key={index}>
+          <span>↑</span>
```

Comment:
This one should also be another arrow but I'm not sure if @jorisleker put this somewhere on the figma :')

Reply:
Added all keys we use now [on the components page](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=5270-12307&node-type=frame&t=eUOXw7n8DKYju3x5-11)

Reply:
The up and down arrows are unicode chars in our default body font